components/CMSUnderConstructionPage.js

```diff
@@ -0,0 +1,57 @@
+import styled from 'styled-components';
+import { formatDate } from '../utility/firebase';
+import nwplusLogo from '../assets/nwplus.svg';
+
+const NwplusImage = styled.img`
```

Comment:
hahaha we can't be upset at non-club members calling us NwPlus or Nwplus or nwplUS anymore

Reply:
it has to be capital because its a styled component :(

Reply:
NwPlus at leeeast